src/components/content-tab-settings.tsx

```diff
@@ -105,6 +106,7 @@ export function ContentTabSettings( { selectedSite }: ContentTabSettingsProps )
 				</tbody>
 			</table>
 			<DeleteSite />
+			<CliExecuteButton />
```

Comment:
I know this button was created for demo/debugging purpose, let's remove it when we add the tests.